layer/filestore_unix.go

```diff
@@ -0,0 +1,13 @@
+// +build !windows
```

Comment:
Also, this might be useful for future "LCOS" and "LCOF", if anyone wants to implement daemon for Solaris and FreeBSD